### PR TITLE
add more perfcounter for MOSDOp

### PR DESCRIPTION
**Pull request by majianpeng**

### CODE REVIEW

File: src/msg/async/Stack.h [hunk 1]
```diff
@@ -193,6 +193,7 @@ enum {
   l_msgr_send_bytes,
   l_msgr_created_connections,
   l_msgr_active_connections,
+  l_msgr_osdop_lifetime_lat,
```
Comment (yuyuyu101) [inline src/msg/async/Stack.h]: it's a little confusing to put osd op latency here, could we delegate logger in MOSDOpReply Message?

Comment (majianpeng) [inline src/msg/async/Stack.h]: But this make  asyn include osd.h which to know l_osd_*.

**Comment by liewegas** [comment]
needs rebase (or close)?

**Comment by majianpeng** [comment]
I'll close it.

**Pull request closed without merging**